tools/panaroo/.shed.yml

```
@@ -0,0 +1,10 @@
+name: panaroo
+owner: galaxy-australia
+categories:
+- Pangenome
```

IUC doesnt have a Pangenome catogery yet
```suggestion
- Sequence Analysis
```

---

That's correct ! Galaxy does not have Pangenome yet.